portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,42 +26,59 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private List<String> messages;
+    
+    public class Name 
+    {
+        private ArrayList<String> names = new ArrayList<String>();
+    }
   
-  @Override
-  public void init() 
-  {
-    messages = new ArrayList<>();
-    messages.add("This is message one.");
-    messages.add("This is messsage two! Welcome to the party json!");
-    messages.add("This is messsage three! This is too easy...");
-  }
+    private Name name = new Name();
```

Comment:
Where is this used?

Reply:
The .toJson function takes an object as a parameter and easily converts it to JSON. This way I do not have to hard code the JSON file. I created the class of Name to do this. The private variable in the Name class is an array that shows up in my JSON file. The array is accessed in the get names function. Is there an easier way to get something in JSON format?

Reply:
I see now. I guess I was just surprised it worked since the inner property is marked private. Anyway, still approved. Gonna merge this.